src/widgetastic_patternfly/__init__.py

```diff
@@ -2042,7 +2079,7 @@ def click_location(self, name, handle_alert=True):
         try:
             location = next(loc for loc in self._path_elements if br.text(loc) == name)
         except StopIteration:
-            self.logger.exception(f'Given location name [{name}] not found')
```

Comment:
Why remove the use of f-string here? I see you didn't use it above, which is fine, but I don't think its appropriate to remove uses of it without some reason behind it.

Reply:
Ignore me, this was working around the py3.5 support that was still active in master, which has been removed.